src/pages/login/index.tsx

```diff
@@ -21,6 +30,12 @@ function Login() {
 
 export default Login;
 
+const AccountCrop = styled.div`
```

Comment:
저희 Styled로 시작해서

```suggestion

const StyledAccountCrop = styled.div`
```

로 수정해주시면 좋을 것 같아요